packages/firestore/src/lite-api/aggregate.ts

```diff
@@ -36,15 +36,15 @@ import { LiteUserDataWriter } from './reference_impl';
 import { fieldPathFromArgument } from './user_data_reader';
 
 /**
- * Calculates the number of documents in the result set of the given query,
+ * Calculates the number of documents in the result set of the given query
  * without actually downloading the documents.
  *
  * Using this function to count the documents is efficient because only the
- * final count, not the documents' data, is downloaded. This function can even
+ * final count, not the documents' data, is downloaded. This function can
  * count the documents if the result set would be prohibitively large to
```

Comment:
Suggest "count the documents **in cases where** the result set would be prohibitively large to download entirely (thousands of documents)."

https://github.com/firebase/firebase-js-sdk/pull/6608#discussion_r985997839

I think this phrasing appears elsewhere.